tfjs-core/src/ops/browser_async.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import { Tensor3D } from '../tensor';
+import { PixelData } from '../types';
+import { fromPixels } from './browser';
```

Comment:
Shouldn't you add a `fromPixelsAsync` in `browser` and import here?

Reply:
I take another look. Maybe the best way is to directly add `export const fromPixelsAsync = op({fromPixelsAsync_});` in browser.ts. Then `browser_async.ts` can be removed.